addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -262,5 +261,18 @@
     <t t-out="'&#847; &#8203; ' * 140"/>
 </div>
         </template>
+
+        <template id="mail_notification_responsible_signature_layout"
+                  name="Mail: mail notification layout with responsible signature (user_id of the record) instead of the author template"
```

Comment:
Would simplify a bit :D

Reply:
I have removed the last part " instead of the author template" and kept the rest.